apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.ejs

```diff
@@ -8,11 +8,22 @@
   </head>
   <body>
     <script>
-      $(function () {
-          $('[data-toggle="popover"]').popover({sanitize: false})
+      $(() => {
+        $('[data-toggle="popover"]').popover({ sanitize: false });
+
+        // Prevent the dropdown menu from closing when the popover is opened.
+        $('.js-mark-variant-as-broken[data-toggle="popover"]').on("click", (e) => {
+          e.stopPropagation();
+        });
+
+        // Hide all popovers when the dropdown menu is closed.
+        $('.js-question-actions').on('hide.bs.dropdown', () => {
+          $('.js-mark-variant-as-broken[data-toggle="popover"]').popover("hide");
+        });
```

Comment:
Is there any way to implement a more generic version of the above code for all pages? I haven't thought about this carefully but it just feels like the kind of thing we are going to have to replicate across many pages.

Reply:
IMO popovers launching out of a dropdown isn't something we necessarily want to encourage, so I'd rather not make it even easier to do that. Can we keep this local to this page for now?

Reply:
I removed the popover-from-dropdown in 7e0b69c and 66edad4; we now use modals for confirmation.

Reply:
Excellent! 🙏